src/inbox/mod_inbox.erl

```diff
@@ -81,8 +90,8 @@ start(Host, Opts) ->
     IQDisc = gen_mod:get_opt(iqdisc, Opts, no_queue),
     MucTypes = get_groupchat_types(Host),
     lists:member(muc, MucTypes) andalso mod_inbox_muc:start(Host),
-    ejabberd_hooks:add(user_send_packet, Host, ?MODULE, user_send_packet, 90),
-    ejabberd_hooks:add(filter_local_packet, Host, ?MODULE, filter_packet, 90),
+    ejabberd_hooks:add(user_send_packet, Host, ?MODULE, user_send_packet, 89),
```

Comment:
If there's a strict ordering requirement for these handlers (and changing 90 -> 89 suggests so) it should be stated, e.g. as a comment here.